lib/winston/config.js

```diff
@@ -7,6 +7,8 @@
  */
 
 var colors = require('colors/safe');
+var supportsColor = require('colors/lib/system/supports-colors');
```

Comment:
@christophehurpeau I think there's an easier way to `require` that: https://github.com/Marak/colors.js/blob/5a2773dbec7e9f3195df692a0c3dacbef7065b6d/lib/colors.js#L39